docs/_static/custom.js

```diff
@@ -81,18 +92,11 @@ document.addEventListener('DOMContentLoaded', () => {
     parent.insertBefore(table, element.nextSibling);
   });
 
-  Object.entries(settings).forEach(([name, setter]) => {
+  Object.entries(settings).forEach(([name,]) => {
```

Comment:
In this case you'd be better of with `for (const key of Object.keys(settings))`.